### PR TITLE
Validate token

### PR DESCRIPTION
**Pull request by sauljabin**

### CODE REVIEW

File: local-dev/setup.sh [hunk 1]
```diff
@@ -79,13 +74,18 @@ EOF
     KEYCLOAK_ADMIN_PASSWORD="admin"
     KEYCLOAK_PORT="8888"
 
-    KEYCLOAK_ADMIN_ACCESS_TOKEN=$(http -q --form "http://localhost:${KEYCLOAK_PORT}/realms/master/protocol/openid-connect/token" \
```
Comment (sauljabin) [inline local-dev/setup.sh:82]: -q  is a bug @MateoRojas check that in your script

File: local-dev/setup.sh [hunk 2]
```diff
@@ -79,13 +74,18 @@ EOF
...
+    done
+
+    KEYCLOAK_ADMIN_ACCESS_TOKEN=$(http --form "http://localhost:${KEYCLOAK_PORT}/realms/master/protocol/openid-connect/token" \
         client_id=admin-cli \
         username="$KEYCLOAK_ADMIN" \
         password="$KEYCLOAK_ADMIN_PASSWORD" \
         grant_type=password | jq -r ".access_token")
 
-    http -q -A bearer -a "$KEYCLOAK_ADMIN_ACCESS_TOKEN" "http://localhost:${KEYCLOAK_PORT}/admin/realms" \
+    http -q -A bearer -a "$KEYCLOAK_ADMIN_ACCESS_TOKEN" POST "http://localhost:${KEYCLOAK_PORT}/admin/realms" \
```
Comment (sauljabin) [inline local-dev/setup.sh:88]: POST is for making it explicit but not necessary

**Comment by sauljabin** [comment]
With this PR @MateoRojas and I cover more types of response. Keyclock could response:
```
{
    "active": false
}
```

So we made the token status more flexible

**Review by coltmcnealy-lh — APPROVED** [review]

**Comment by coltmcnealy-lh** [comment]
Approved, but to be honest this is the part of the code that I understand the least.

**Pull request merged**